maybe/maybe_unix.go

```diff
@@ -0,0 +1,19 @@
+package maybe
+
+import (
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/google/renameio"
+)
+
+// WriteFile mirrors ioutil.WriteFile. On Linux it replaces an existing file
+// with the same name atomically. On Windows the file is not written atomically
```

Comment:
How about:

On Linux it uses renameio.WriteFile to replace…

Reply:
Done.